backend/deploy.js

```diff
@@ -1,6 +1,6 @@
 var crypter = require("../util/crypter.js")
 var spawn = require('child_process').spawn;
-
+var socketHandler = require('./socketHandler.js')
```

Comment:
now in the first parameter, you have to pass a socket object. I don't want to create as a separate function to handle generation. so I made a module named socket handler. it'll handle if the first parameter is socket otherwise it'll just ignore the object do nothing. we'll get a modular code